php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -866,16 +866,19 @@ abstract class NDB_BVL_Instrument extends NDB_Page
                 'Window Difference (+/- Days)'
             );
         }
-        $examiners = $this->_getExaminerNames();
-        $this->addSelect('Examiner', 'Examiner', $examiners);
+
+        if (isset($this->commentID)) {
```

Comment:
`if (empty($this->getCommentID()) {` would work with empty string and is using the getter (which is probably there for something... or not)
But this mean that the Examiner drop down wont be shown which is not ideal. It would make more sense to have `$this->_getExaminerNames()` return an empty array when there is no commentid.